tensortrade/env/default/observers.py

```diff
@@ -43,7 +44,14 @@ def _create_wallet_source(wallet: 'Wallet', include_worth: bool = True) -> 'List
 
         if include_worth:
             price = Stream.select(wallet.exchange.streams(), lambda node: node.name.endswith(symbol))
-            worth = (price * total_balance).rename("worth")
+
+            if price.value is None:
+                price.value = price.forward()
+
+            if total_balance.value is None:
+                total_balance.value = total_balance.forward()
+
+            worth = mul(price, total_balance).rename('worth')
             streams += [worth]
```

Comment:
I feel like price should never be none and earlier in the process there should be a fill forward done on the original dataset so raising an error might be more preferable. @mwbrulhardt  could you please chime in?

Reply:
Also if we forward() here will that end up being out of sync?

Reply:
Yea a price should never be`None` coming into this context. I believe I have unit tests that check this function.

Reply:
The user in the issue uses a custom downloader for data which probably contains errors or no ffil() etc...and so therefore I suggest we go back to him for a cleaner test case.